integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -48,6 +49,9 @@ function show_help() {
     echo "  --cpu                      Comma separated list of CPU architectures. Like arm or arm64"
     echo "  --tizen-sdk-path           Set directory for Tizen SDK installation. Default is $TIZEN_SDK_ROOT"
     echo "  --tizen-sdk-data-path      Set directory for Tizen SDK runtime data. Default is $TIZEN_SDK_DATA_PATH"
+    echo "  --tizen-pkgs-dir           Set the directory to store downloaded Tizen SDK packages, default is /tmp/XXXXXX"
+    echo "  --fetch-sdk-pkgs           Fetch RPMs and ZIPs from https://download.tizen.org/ into pkgs path"
+    echo "  --install-sdk-pkgs         Install fetched packages into Tizen SDK directory (default $TIZEN_SDK_ROOT)"
```

Comment:
How about fetching and always storing packages in the cache dir (maybe a subdir in `~/.cache` or some other common persistent "cache" directory, just like `~/.cache/pip`). Then, when package is already in the cache skip the fetch stage (for a single package). The install stage should always happen (the name of this script is `install.sh`). Optionally you might add an option to always-fetch even though a package is in the cache already (alternatively one can simply remove the cache dir to achieve the same result).